Building Applications with React and Flux.md

```diff
@@ -0,0 +1,15 @@
+# This is a development environment for React
+
+#### With React comming to Free Code camp soon. I thought this would be a aset to the community.
+
+ > This is designed to run in the cloud nine environment.
+> Once this is installed in a custom environment at cloud nine. All you have to do is type npm install from the bash terminal.
```

Comment:
Within a wiki it is best not to use words such as 'you'. "Once installed, enter `npm install` into the bash terminal."